proto/viam/app/data/v1/data.proto

```diff
@@ -110,5 +110,5 @@ message BinaryDataByIDsRequest {
 message BinaryDataByIDsResponse {
```

Comment:
Supernit: could you add comments for BinaryDataByIDsResponse and BinaryDataByFilterResponse for consistency?

Reply:
yup!